File: Programmers/Lv0/배열만들기5.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> intStrs, int k, int s, int l) {
    vector<int> answer;
    for (int i = 0; i < intStrs.size(); i++) {
        int num = 0;
        for (int j = s; j < s + l; j++) {
            num *= 10;
            num += intStrs[i][j] - '0';
        }
        if (num > k)
            answer.push_back(num);
    }
    return answer;
}

File: Programmers/Lv0/배열만들기5.py
def solution(intStrs, k, s, l):
    answer = []
    for str in intStrs:
        if int(str[s:s + l]) > k:
            answer.append(int(str[s:s + l]))
    return answer

# 리스트 컴프리헨션
def solution(intStrs, k, s, l):
    return [int(str[s:s + l]) for str in intStrs if int(str[s:s + l]) > k]